tests/unit/src/utils/test_rlp.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import pytest_asyncio
+from rlp import decode, encode
+
+# cannot parametrize and use random data at the same time
```

Comment:
I don't understand why you can't parametrize on `data_len` and use
```
random.seed(data_len)
random.getrandbits(data_len * 8)
```
inside a test
Ju